### PR TITLE
Allow a reason to be given for the 'it should work' step

### PR DESCRIPTION
**Pull request by makmic**

### CODE REVIEW

File: lib/spreewald/development_steps.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # coding: UTF-8
 
-# Marks scenario as pending
-Then /^it should work$/ do
+# Marks scenario as pending, optionally explained with a reason.
+Then /^it should work(?:.+?)?$/ do
```
Comment (makmic) [inline lib/spreewald/development_steps.rb]: Without the outer lazy operator `?`, the reason would no longer be optional. Omitting the inner one results in a warning:

```
warning: nested repeat operator '+' and '?' was replaced with '*' in regular expression: /^it should work(?:.+)?$/`
```

**Comment by judithroth** [comment]
We talked about adding some simple tests and displaying the reason with `pending('message')`. Is there anything else I can help you with?

**Comment by makmic** [comment]
I think I'll be fine, thank you! I expect to find the time for implementing these changes within a week.

**Comment by makmic** [comment]
Hi @judithroth,
I added the requested (although quite minimalistic) tests with this commit: https://github.com/makandra/spreewald/pull/86/commits/166fbfce21a6d86375303b538b69e4f42ab3c5b2

Something I want to point out: We configured the Cucumber 3 tests with the `--strict` flag. I needed to add the `---no-strict-pending` flag to prevent the test suite from failing:
  
spreewald_pending_failure

**Pull request merged**